pkg/datapath/linux/config/config.go

```diff
@@ -211,6 +211,8 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 	cDefinesMap["CT_REPORT_INTERVAL"] = fmt.Sprintf("%d", int64(option.Config.MonitorAggregationInterval.Seconds()))
 	cDefinesMap["CT_REPORT_FLAGS"] = fmt.Sprintf("%#04x", int64(option.Config.MonitorAggregationFlags))
 	cDefinesMap["RATELIMIT_MAP"] = "cilium_ratelimit"
+	cDefinesMap["RATELIMIT_ID_ICMPV6"] = "1"
+	cDefinesMap["RATELIMIT_ID_EVENTS_MAP"] = "2"
```

Comment:
nit: you can define ratelimit keys using iota instead of directly using "1" and "2" (magic numbers)

i.e.
```
const (
  RATELIMIT_ID_ICMPV6 = iota
  RATELIMIT_ID_EVENTS_MAP
)

// ... and then
cDefinesMap["RATELIMIT_ID_ICMPV6"] = fmp.Sprintf("%d", RATELIMIT_ID_ICMPV6) 
cDefinesMap["RATELIMIT_ID_EVENTS_MAP"] = fmp.Sprintf("%d", RATELIMIT_ID_EVENTS_MAP) 

```